package.json

```diff
@@ -47,18 +47,18 @@
     "expo-permissions": "13.2.0",
     "expo-splash-screen": "0.15.1",
     "expo-status-bar": "1.3.0",
-    "expo-updates": "0.13.2",
+    "expo-updates": "~0.13.3",
```

Comment:
renovate does always pin the dependencies instead of using the ` ~`. I don't know the advantages and disadvantages of either. I guess like this is also fine?

Reply:
yes we should remove `~` and `^` because we use expo's updating compatible versions which needs exact version numbers.